chn_translation_arima_models.Rmd

```diff
@@ -0,0 +1,190 @@
+# Chinese translation of using ARIMA models in R
+
+Kehao Guo
+
+Source: [link](https://a-little-book-of-r-for-time-series.readthedocs.io/en/latest/src/timeseries.html#arima-models)
+
+## ARIMA模型
+
+指数平滑方法对于预测很有用，并且不对时间序列中的相邻数据作假设。但是，如果你想用指数平滑方法作预测区间，预测区间会需要不相关的、正态分布的、零均值化的且水平方差的预测误差。
+
+虽然指数平滑方法不对时间序列中的相邻数据作假设，在某些情况下你可以通过考虑数据间的相关性来做出更好的预测模型。差分整合移动平均自回归模型（ARIMA）包含了一个允许异常成分中非零自相关性的统计模型。
+
+### 差异化一组时间序列
+
+ARIMA模型是为平稳时间序列定义的。所以如果需要处理一组非平稳时间序列，你首先需要‘差异化’时间序列直至获得一组平稳时间序列。如果你需要差异化d次来获得平稳时间序列，那你就会用ARIMA(p,d,q)模型，d代表差异次数。
+
+你可以在R中使用‘diff()’函数来差异化时间序列。比如，1866年至1911年女士裙子的直径均值上下浮动且不平稳：
+
+![](pics/1.png)

```

Comment:
The location of the folder and file is incorrect. Other markdown links have the same issue.